zipkin-storage/cassandra/src/main/java/zipkin2/storage/cassandra/SelectTraceIdsFromSpan.java

```diff
@@ -104,8 +104,7 @@
       Input input =
           new AutoValue_SelectTraceIdsFromSpan_Input(
               serviceName,
-              // % for like, bracing with ░ to ensure no accidental substring match
```

Comment:
we could do something like detect if the indexing type is updated. Ex if the mode=CONTAINS then box annotationKey like before?